src/components/ActionBar.vue

```diff
@@ -0,0 +1,83 @@
+<script setup>
+import { ref, computed } from "vue";
+import useMovieStore from "../store/movieStore";
+
+const movieStore = useMovieStore();
+
+const sortByDate = ref(true);
+
+const moviesCount = computed(() => {
+    if (movieStore.list) {
+        return movieStore.list.length;
+    }
+    else return 0;
+});
+
+const sortHandler = (option) => {
+    if (option === "releaseDate") sortByDate.value = true;
+    else if (option === "imdbRating") sortByDate.value = false;
+    else sortByDate.value = true;
+    movieStore.setSortByParam(option);
+};
+</script>
+
+<template>
+    <div class="panel">
+        <div v-if="movieStore.isMovieDetailOpen" class="panel__moviesCount">Films by {{ movieStore.getMovieById.genres[0] }} genre</div>
+        <div v-else class="panel__moviesCount">{{ moviesCount.valueOf() }} movies found</div>
+        <div v-if="!movieStore.isMovieDetailOpen" class="panel__sort">
+            <div class="panel__sort__heading">SORT BY</div>
+            <div class="panel__sort__options">
+                <p @click="sortHandler('releaseDate')" v-bind:class="{ 'panel__sort__selected': sortByDate }">RELEASE DATE
```

Comment:
You can't use element in element, look for BEM rules.
You don't have to use the `v-bind` thing:

```html
<p class="{ 'panel__sort--selected': sortByDate }" @click="sortHandler('releaseDate')">RELEASE DATE</p>
```